src/features/deeplinks/routing.ts

```diff
@@ -31,6 +31,9 @@ export const DEEPLINK_TO_SCREEN_CONFIGURATION: DeepLinksToScreenConfiguration =
     }
     return { screen: 'Home', params: { shouldDisplayLoginModal: false } }
   },
+  offer: function (params) {
+    return { screen: 'Offer', params: { id: params ? params.id : '' } }
+  },
```

Comment:
Tu peux ajouter des tests dans `routing.test.ts` ?
D'ailleurs si tu peux trouver le moyen qu'il ne soit pas possible d'ajouter une propriété dans `DEEPLINK_TO_SCREEN_CONFIGURATION` sans la tester dans `routing.test.tsx` you're welcome.

Reply:
la couverture serait utile pour ça, mais à part ça, je vois pas trop comment checker que toutes les lignes soient testées...

Reply:
du coup il se passe quoi si on appelle le deeplink sans id ?

Reply:
On va sur la page offer detail, mais y'a pas d'id. Je pensais faire la gestion des ids vides à ce moment là, vu qu'on fait la requetes sur cette page. Pareil avec une id invalide